dotnet-desktop-guide/framework/winforms/controls/how-to-show-a-font-list-with-the-fontdialog-component.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Show a Font List with the FontDialog Component"
+description: Learn how to show a font list with the FontDialog component, which allows users to select a font, as well as change its display aspects, such as its weight and size.
```

Comment:
This one's over the 160 character limit. I'd shorten it to: "Learn how to show a font list with the FontDialog component, which allows users to select a font, as well as change its display aspects."

Reply:
Addressed.